src/main/java/io/dropwizard/elasticsearch/health/EsIndexDocsHealthCheck.java

```diff
@@ -1,113 +0,0 @@
-package io.dropwizard.elasticsearch.health;
-
-import com.codahale.metrics.health.HealthCheck;
-import com.google.common.collect.ImmutableList;
-import org.elasticsearch.action.admin.indices.stats.IndexStats;
-import org.elasticsearch.action.admin.indices.stats.IndicesStatsResponse;
-import org.elasticsearch.client.Client;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.base.Preconditions.checkNotNull;
-
-/**
- * A {@link HealthCheck} which checks if one or more indices in Elasticsearch contain a given number of documents
- * in their primaries.
- *
- * @see <a href="http://www.elasticsearch.org/guide/reference/api/admin-indices-stats/">Admin Indices Stats</a>
- */
-public class EsIndexDocsHealthCheck extends HealthCheck {
```

Comment:
Could we migrate the health check instead of deleting it?

Reply:
brought it back and migrated. Still a bit wondering whether the docCount check is a useful health check though...